src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
+@import "./blocks/page.scss";
+@import "./blocks/header.scss";
+@import "./blocks/container.scss";
+@import "./blocks/cards.scss";
+@import "./blocks/stars.scss";
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
 }
```

Comment:
for these types of changes, you should use a separate file (normalize.scss)

Reply:
yes, you are right